### PR TITLE
fix `[sriov] Bond CNI integration bond cni over sriov` test

### PR DESCRIPTION
**Pull request by zeeke**
This PR should fix the following test error:

```
/remote-source/app/cnf-tests/testsuites/e2esuite/bond/bond_sriov.go:64
Unexpected error:
    <*errors.StatusError | 0xc0021d4c80>: {
        ErrStatus: {
            TypeMeta: {Kind: "", APIVersion: ""},
            ListMeta: {
                SelfLink: "",
                ResourceVersion: "",
                Continue: "",
                RemainingItemCount: nil,
            },
            Status: "Failure",
            Message: "admission webhook \"network-resources-injector-config.k8s.io\" denied the request: could not find network attachment definition 'bond-testing/bond': could not get Network Attachment Definition bond-testing/bond: the server could not find the requested resource",
            Reason: "",
            Details: nil,
            Code: 400,
        },
    }
    admission webhook "network-resources-injector-config.k8s.io" denied the request: could not find network attachment definition 'bond-testing/bond': could not get Network Attachment Definition bond-testing/bond: the server could not find the requested resource
occurred
/remote-source/app/cnf-tests/testsuites/e2esuite/bond/bond_sriov.go:77
```

Alongside, I added two commits to improve the around `namespaces.CleanPods(...)` and `networks.CleanSriov(...)`.

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/bond/bond_sriov.go [hunk 1]
```diff
@@ -32,63 +31,69 @@ var _ = Describe("[sriov] Bond CNI integration", func() {
 	apiclient := client.New("")
 
 	execute.BeforeAll(func() {
+		if discovery.Enabled() {
+			Skip("Tuned sriov tests disabled for discovery mode")
```
Comment (mmirecki) [inline cnf-tests/testsuites/e2esuite/bond/bond_sriov.go:35]: This should go into `BeforeEach`
`BeforeAll` will Skip only the first Context

Comment (mmirecki) [inline cnf-tests/testsuites/e2esuite/bond/bond_sriov.go:35]: This should go into `BeforeEach`

Comment (zeeke) [reply cnf-tests/testsuites/e2esuite/bond/bond_sriov.go:35]: right! fixing ...

**Pull request merged**